libs/geography/testing/src/factories/country.factory.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import * as faker from 'faker/locale/en_US';
+
+import { DaffCountry } from '@daffodil/geography';
+
+import { DaffModelFactory } from '@daffodil/core/testing';
+
+export class MockCountry implements DaffCountry {
+  id = faker.random.number(1000);
```

Comment:
id is a string on the type. Either cast this or expand the type. It may be preferable to expand the type, as we can't say what it will explicitly be for every platform.

Reply:
Changed